homeassistant/components/webhook/__init__.py

```diff
@@ -102,13 +107,16 @@ async def async_handle_webhook(
         )
         # Look at content to provide some context for received webhook
         # Limit to 64 chars to avoid flooding the log
-        content = await request.content.read(64)
+        content = await content_stream.read(64)
         _LOGGER.debug("%s", content)
         return Response(status=HTTPStatus.OK)
 
     if webhook["local_only"]:
+        if TYPE_CHECKING:
+            assert isinstance(request, Request)
+            assert request.remote is not None
```

Comment:
I am not sure if this is the best way to handle it here. I decided to use `if TYPE_CHECKING` to avoid performance hits, but maybe there is a better way?

Reply:
I think it's ok for now.